src/containers/LoginPage/index.js

```diff
@@ -1,5 +1,47 @@
-import React from 'react'
+import React, { Component } from 'react'
+import axios from 'axios';
+import View from './View';
 
-export default function LoginPage() {
-  return <p>Loginpage</p>
+export default class LoginPage extends Component {
+  constructor() {
+    super();
+    this.state = {
+      username: '',
+      password: '',
+      isLogin: false
+    };
+  }
+
+  onChangeUsername = (e) => {
+    this.setState({
+      username: e.target.value
+    });
+  }
+
+  onChangePassword = (e) => {
+    this.setState({
+      password: e.target.value
+    });
+  }
+
+  doLogin = (e) => {
```

Comment:
```suggestion
  handleLogin = (e) => {
```
kiểu đặt tên sẽ như này nhé:
- với event handler mình viết sẽ là handle + action. eg: handleLogin, handleSubmit, ...
- với prop thì sẽ đặt là on + action: onLogin, onSubmit